rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -239,7 +239,7 @@ dfs without knn vector profiling:
         index: keywords
         body:
           settings:
-            index.number_of_shards: 1
+            index.number_of_shards: 2
```

Comment:
It did not :/

Reply:
With kNN search, we do DFS even when there is only 1 shard! We have an optimization planned to avoid this: https://github.com/elastic/elasticsearch/issues/88921.

Reply:
This was testing for the dfs profiling **without** a knn query to ensure we don't get the knn portion of the dfs profile results.

Reply:
Oh I see, thanks for clarifying.